provisioning/transport/http/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "azure-iot-provisioning-http",
+  "version": "1.0.0",
```

Comment:
>1.0.0 [](start = 14, length = 5)

Since we're shooting for public preview before release, i'd set this to 0.0.1 right away (just to avoid publishing something by mistake)